sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -1931,7 +1933,7 @@ def __init__(self, **kwargs):
         self.domain_filter = kwargs.get("domain_filter", None)
         self.categories_filter = kwargs.get("categories_filter", None)
         self.string_index_type = kwargs.get("string_index_type", "UnicodeCodePoint")
-        self.disable_service_logs = kwargs.get("disable_service_logs", False)
+        self.disable_service_logs = kwargs.get("disable_service_logs", True)
```

Comment:
had the wrong default here :(

Reply:
how are defaults working in Python? if the user doesn't set them, the library sets them and sends them to the service? or do u leave it up to the service?

I am asking because in .NET we do the second option.

Reply:
Sorry I missed this. I prefer to do the latter. Looking across TA we have a mix of both. Opening an issue to look at this and make sure any changes we make don't cause unintended behavior. https://github.com/Azure/azure-sdk-for-python/issues/19939

Reply:
Do you mean across TA Python code? or across all languages.

Reply:
I only looked at TA Python code.

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/TextAnalytics/stable/v3.1/TextAnalytics.json#L811

In the swagger, it is default to `True`. If user doesn't set anything, do we expect the PII Task to use the default `true`?

Reply:
>do we expect the PII Task to use the default true

Yes. If the user or us don't pass anything, the service will use a default value of true. 
And ideally, we should do that. Not pass anything and let the user manage the defaults